### DISCUSSION TITLE
Have to say, I'm enjoying v2.0.0

### DISCUSSION THREAD

**Discussion by mendhak** [11ty/eleventy#2759] [Uncategorized]
Congrats on the release everyone. I'm developing my blog with 11ty at the moment so I thought I might as well try an upgrade and see what's broken.    

The upgrade helper worked well and told me what to pay attention to, and there wasn't a huge amount.  But the best bit is this new Html Base Plugin.  My post files are much cleaner now, no more `| url` in many places, no more concatenation needed in shortcodes and filters... so I'm going around and deleting lots of little bits of cruft I had accumulated. 

I am a simple man.  If I get to delete something, I produce endorphins.   Many thanks for this.

**Comment by zachleat** [comment]
Thank you @mendhak!! Really glad to hear this feedback 🙌🏻
